netci.groovy

```diff
@@ -219,6 +219,8 @@ commitPullList.each { isPr ->
       def myJob = job(jobName) {
         description("Windows ${configuration} tests on ${buildTarget}")
         steps {
+          batchfile('tasklist /FI "IMAGENAME eq devenv.exe"')
+          batchfile('taskkill /FI "IMAGENAME eq devenv.exe"')
```

Comment:
As per offline discussion, this was moved into the finally clause of the build.ps1 script (and only executes if `cibuild` is true).